subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -815,6 +815,13 @@ config BT_CTLR_USER_CPR_INTERVAL_MIN
 	  When enabled, controller will accept Connection Parameter Request
 	  intervals down to a proprietary minimum value.
 
+config BT_CTLR_USER_CPR_ANCHOR_POINT_MOVE
+	bool "Proprietary handling of anchor point movement"
```

Comment:
No longer proprietary...

Reply:
Well, the USER_EXT is still referred to as 'proprietary extensions in controller', so I guess using proprietary term here is as valid as for the other ones?